src/translations/components/TranslationsAttributesPage/messages.ts

```diff
@@ -0,0 +1,15 @@
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
```

Comment:
Should be a named export `transtionsAttributesPageFieldsMessages` and in the file imported 
```
import { transtionsAttributesPageFieldsMessages as messages } from 'messages'